fastlane/Fastfile

```diff
@@ -52,3 +92,39 @@ def run_flutter_command(command, subdir = '.')
     sh "fvm flutter #{ command }"
   end
 end
+
+def increment_version! term
+  pubspec = PubSpec.new '../pubspec.yaml'
+  version = Semantic::Version.new pubspec.version
+  new_version = version.increment!(term)
+  new_version.build = version.build
+  pubspec.version = new_version.to_s
+  pubspec.write!
+end
+
+def increment_build_number!
+  pubspec = PubSpec.new '../pubspec.yaml'
+  version = Semantic::Version.new pubspec.version
+  version.build = (version.build.to_i + 1).to_s
+  pubspec.version = version.to_s
+  pubspec.write!
+end
+
+class PubSpec
+  def initialize path
+    @path = path
+    @content = YAML.load_file(@path)
+  end
+
+  def write!
+    File.open(@path, "w") { |file| file.write(@content.to_yaml) }
```

Comment:
Writing `YAML` will mess up formatting.
Should just use `File` and `@content.gsub(...`.